### PR TITLE
Refactor: Use pest for tests

### PR DESCRIPTION
**Pull request by bpolaszek**
Depends on #13.

Ping @simPod

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ on:
     - cron: "0 0 1 * *"
 
 jobs:
-  phpunit:
-    name: "PHPUnit"
+  unit-tests:
```
Comment (simPod) [inline .github/workflows/continuous-integration.yml:10]: I'd name it `pest`, it's more specific

File: .scrutinizer.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+language: php
```
Comment (simPod) [inline .scrutinizer.yml]: I guess we can not add this. Or is there any benefit from it? I considered scrutinizer obsolete for some year now

**Review by simPod — APPROVED** [review]

File: .scrutinizer.yml (hunk 2 above)
Comment (bpolaszek) [reply .scrutinizer.yml]: I used to use it for the "code quality" insight + badge, but there's no real benefit actually. Will remove it.

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (bpolaszek) [reply .github/workflows/continuous-integration.yml:10]: _Too_ specific IMO 🙂 when the unit test framework changed, I had to change the name, tho the task remained the same.

File: README.md [hunk 3]
```diff
@@ -187,7 +187,7 @@ require_once '/path/to/this/library/src/iterable-functions.php';
 Unit tests
 ==========
 ```
-./vendor/bin/phpunit
+php vendor/bin/pest
```
Comment (simPod) [inline README.md:190]: is the php needed here? just curious

Comment (bpolaszek) [reply README.md:190]: It's more a habit I have, no biggie

**Pull request merged**